package.json

```diff
@@ -52,7 +52,6 @@
     "/dist"
   ],
   "dependencies": {
-    "cross-fetch": "^3.1.5",
```

Comment:
Oh, did we not actually remove this in the previous PR? https://github.com/pinecone-io/pinecone-ts-client/pull/250/files

Reply:
lol nope